### ISSUE TITLE
OAuth instead of Bot Credentials

### ISSUE THREAD

**Issue by MarcoReidelbach** [MarcoReidelbach/MaRDMO-Plugin#50]
Users should be able to edit the wikibase through a simple authentification instead of some bot credentials.

**Comment by MyPyDavid** [comment]
maybe the code from https://github.com/rdmorganiser/rdmo-plugins-zenodo/blob/main/rdmo_zenodo/exports.py and `from rdmo.services.providers import OauthProviderMixin` could be re-used for this

**Comment by MarcoReidelbach** [comment]
Yes, that is the idea I currently try to follow. I am in contact with the MaRDI staff responsible for the MaRDI Portal (wikibase) to set up a new oauth consumer.

**Comment by MarcoReidelbach** [comment]
OAuth2 procedure integrated, work with test.wikidata.org Waiting for some corrections of MaRDI Portal settings

**Comment by MarcoReidelbach** [comment]
Fully integrated
